include/ck/utility/amd_wmma.hpp

```diff
@@ -41,9 +46,15 @@ struct intrin_wmma_f32_16x16x16_bf16_w32<16, 16>
     template <class FloatC>
     __device__ static void Run(const bhalf16_t& reg_a, const bhalf16_t& reg_b, FloatC& reg_c)
     {
+#if defined(__gfx11__)
         reg_c.template AsType<float8_t>()(Number<0>{}) =
             __builtin_amdgcn_wmma_f32_16x16x16_bf16_w32(
                 reg_a, reg_b, reg_c.template AsType<float8_t>()[Number<0>{}]);
+#else
```

Comment:
This is a false fallback, and it's very deep in the calling stack. If we put it here, I'm concerned this means people would not realize they are making a mistake when trying to use  ```intrin_wmma_f32_16x16x16_f16_w32``` on non gfx11 machine.

As an example, there is no false fallback in mfma intrinsic on gfx9
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/76fcdc60e9ef848d23527079ad59ca6f4040ca59/include/ck/utility/amd_xdlops.hpp#L15-L26

I think for wmma intrinsic, you can do similar things